Tribler/Core/Modules/MetadataStore/OrmBindings/channel_metadata.py

```diff
@@ -455,17 +401,18 @@ def channel_state(self):
             return "Preview"
 
         @db_session
-        def to_simple_dict(self):
+        def to_simple_dict(self, **kwargs):
             """
             Return a basic dictionary with information about the channel.
             """
-            result = super(ChannelMetadata, self).to_simple_dict()
-            result.update({
-                "torrents": self.num_entries,
-                "subscribed": self.subscribed,
-                "votes": self.votes/db.ChannelMetadata.votes_scaling,
-                "state": self.channel_state
-            })
+            result = super(ChannelMetadata, self).to_simple_dict(**kwargs)
+            result.update(
+                {
+                    "state": self.state,
+                    "subscribed": self.subscribed,
+                    "votes": self.votes / db.ChannelMetadata.votes_scaling,
```

Comment:
No, it really can't. There is only one case when this could happen: if the user modifies their SQLite DB with some external tool directly to set it to 0.

Reply:
Thanks, that saves me checking up 👍